reference/api/keys.md

```diff
@@ -163,7 +165,7 @@ Results can be paginated using the `offset` and `limit` query parameters.
         "documents.delete"
       ],
       "indexes": [
-        "products",
+        "products*",
```

Comment:
However, can we change that for this example to be more clear? Like using `prod*`?